Lib/inspect.py

```diff
@@ -1258,8 +1258,8 @@ def convert(name, locals=locals,
                 formatarg=formatarg, formatvalue=formatvalue):
         return formatarg(name) + formatvalue(locals[name])
     specs = []
-    for i in range(len(args)):
-        specs.append(convert(args[i]))
+    for i, arg in enumerate(args):
```

Comment:
Is there a reason to keep the `i` at all here?

Reply:
This pattern can be seen in places in the repository.
For example, there is a similar pattern in the inspect.py module in the Signature class:
```for idx, param in enumerate(parameters):```
where idx is not subsequently used.

The original code also did not use a standard:
 ```for i in args:```

I focused on replacing range(len(args)) with enumerate(args).